config/config.go

```
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/BurntSushi/toml"
+)
+
+// Version of config
+const Version = 1
+
+var (
+	// ErrVerNotCompatible returned when version check failed
+	ErrVerNotCompatible = errors.New("the version of config file is not compatible")
```

Please return the error directly. Don't get into error design too early.